pandas/_libs/tslibs/period.pyx

```diff
@@ -1241,6 +1241,24 @@ cdef class _Period(object):
 
     @property
     def hour(self):
+        """
+        Get hours of a day that a Period falls on.
```

Comment:
Maybe "Get the hour of the day component of a Period."? Since a Period can span multiple days.

Reply:
Yup Period can span multiple days .Its really hard to define accurate sentance ... day conponent of the period sounds more clear.